### PR TITLE
fix: 11060; Added quite many shortcuts from MS3

### PR DESCRIPTION
**Pull request by VanSHOE**
Resolves: #11060 

Follow up of https://github.com/musescore/MuseScore/pull/11144.
This simply adds quite many shortcuts from MS3.
I went through the list of MS3 shorcuts and tried to find as many actions I could find there that were also there in the code of MS4 but missing from the shortcuts list and added them in XML

Several parts could require changes, especially the context
All the actions had notation-opened context, so I added the same in XML as well, but please do verify if that is correct
Do note that not all MS3 shortcuts have been added since action codes for most could not be found by me. but a significant chunk definitely has been added

While verifying the changes, lines may look quite a lot, but the three files essentially have the same changes. (so the changes are essentially tripled)

- [x] I signed [CLA](https://musescore.org/en/cla)
- [x] I made sure the code in the PR follows [the coding rules](https://github.com/musescore/MuseScore/wiki/CodeGuidelines)
- [x] I made sure the code compiles on my machine
- [x] I made sure there are no unnecessary changes in the code
- [x] I made sure the title of the PR reflects the core meaning of the issue you are solving
- [x] I made sure the commit message(s) contain a description and answer the question "Why do those changes fix that particular issue?" or "Why are those changes really necessary as improvements?"
- [x] I made sure the commit message title starts with "fix #424242:" if there is a related issue
- [ ] I created the test (mtest, vtest, script test) to verify the changes I made

### CODE REVIEW

File: src/framework/shortcuts/data/shortcuts.xml [hunk 1]
```diff
@@ -1226,4 +1231,545 @@
...
+    </SC>
+  <SC>
+    <key>edit-style</key>
+    <seq></seq>
+    <ctx>notation-opened</ctx>
+    </SC>
+  <SC>
+    <key>interval-2</key>
+    <seq></seq>
+    </SC>
```
Comment (VanSHOE) [inline src/framework/shortcuts/data/shortcuts.xml:1408]: Somehow I have missed a couple contexts here
I have noted this and will fix this alongside any further requested changes

**Comment by MarcSabatella** [comment]
Of these, Special Characters is the most important from an accessibility and general usability standpoint, especially for people writing in languages other than English where you almost constantly need to access these.  The dialog exists already and can be opened from the Properties panel, but that's definitely far from ideal.  I'm assuming that re-adding a command to simply open the dialog would be not be difficult.

The command to open the symbols palette, like the also-missing commands to open the key signatures and time signatures palettes, is a bit different.  What it really does is open the Master Palette but do so already focused on the correct section.  So it's kind of a special version of the command to open the master palette, and it was probably a bit of a hack.  These individual commands are quite useful, but realistically, if it's hard to implement them in the current framework, a passable substitute would be if the master palette remembered which section you were last viewing.  So if you are dealing in symbols a lot, you open the master palette, change to Symbols (possible in a single keystroke - press "End"), do what you want, then the next time you open the master palette, it's already focused on the symbols section.

**Comment by VanSHOE** [comment]
Update: I believe the action itself exists which is called `show-keys`. However the shortcuts themselves are not picked up while typing. This particular issue might need a rework.

**Comment by MarcSabatella** [comment]
At least in MU3, if a shortcut claimed to be active in text edit mode, then it really was active in text edit mode. People manage to shoot ourselves in the foot with this reasonably often, so I'm perfectly happy if this is no longer possible.

I don't have much of an opinion on whether or not a code change to allow function keys while in edit mode should be a separate PR.  I guess I'd lean towards saying do it separately to maximize the chances of this one being approved sooner, in case there are issues with the code for the other.

**Comment by VanSHOE** [comment]
Now assuming the above PR gets approved, I'll push to set visibility of shortcut to true here

**Comment by VanSHOE** [comment]
Now, assuming the above PR is approved, I'll push (I won't force push as a precaution for easy rollback) to change the visibility of this shortcut to true.

**Comment by VanSHOE** [comment]
I have pushed this, and after seeing the xml, I notice some actions are repeated so I would need to do some extra cleanup as well

**Comment by VanSHOE** [comment]
Please have a look @bkunda @MarcSabatella. 
The shortcut should be working now. PR has also been rebased to latest master

**Comment by MarcSabatella** [comment]
It looks good to me.  I will say, it's surprising to me that we should need to add all these empty shortcuts explicitly just to get things to appear in the list; I'd have assumed a simple code change somewhere would be easier.  In 3.x, the shortcuts.xml file contained only the commands with default shortcuts, and it worked fine.

But I know from the "top staff" command, which had an empty shortcut all along, that it does basically work.  See https://github.com/musescore/MuseScore/issues/11522, though, to be sure it *really* works, as that command has some quirks that I don't know if anyone completely figured out.

**Comment by MarcSabatella** [comment]
I see.  This public/private actions thing is new; it used to be that all actions (aka commands) were the same.  I see now you were specifically advised to use this approach, so, all is well.

**Comment by igorkorsukov** [comment]
@VanSHOE 
There is some kind of problem there, an assert is triggered at the start of the application, it looks like the same shortcut is written twice in the file with different contexts

image

**Comment by VanSHOE** [comment]
@igorkorsukov Also a second doubt I had was that in my project I was also working on auto addition of shortcuts, so would this PR be necessary after that?

**Comment by igorkorsukov** [comment]
@VanSHOE at the moment I have revert back this PR.

>I added here did in fact need to have notation-opened context? (Apart from the ones already present)
We need to investigate each shortcut. For example, for `pitch-up`, the context should be `notation-staff-not-tab`, because another action should be performed for the tab staff

**Comment by igorkorsukov** [comment]
> @igorkorsukov Also a second doubt I had was that in my project I was also working on auto addition and categorisation of shortcuts, so would this PR be necessary after that?

In fact, probably after your work this PR will not be needed

**Pull request merged**